src/__tests__/__snapshots__/AnchoredOverlay.test.tsx.snap

```diff
@@ -193,6 +194,7 @@ exports[`AnchoredOverlay should render consistently when open 1`] = `
     font-family="normal"
   >
     <button
+      aria-expanded="true"
```

Comment:
Wierd that this is a string though

Reply:
yeah! 

the types are accepts both, so that's nice

`'aria-expanded'?: boolean | 'false' | 'true';` ref: [@types/react](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v16/index.d.ts#L1624)